src/applications/personalization/profile-2/components/Verified.jsx

```diff
@@ -6,10 +6,15 @@ import PropTypes from 'prop-types';
  * children content
  */
 const Verified = ({ children }) => (
-  <p className="vads-u-margin--0 vads-u-padding-left--3">
-    <i className="fa fa-check vads-u-color--green vads-u-margin-right--0p5 vads-u-margin-left--neg3" />{' '}
-    {children}
-  </p>
+  <span className="vads-u-display--flex">
+    <i
+      className="fa fa-check vads-u-color--green vads-u-margin-top--0p5"
+      aria-hidden="true"
```

Comment:
Along with `aria-hidden="true"` here, would it be possible to add `role="img"` or `role="presentation"` here?

Reply:
Definitely! Since I had already merged this PR, I added this new role to the changes in [this other, unrelated PR ](https://github.com/department-of-veterans-affairs/vets-website/pull/13562)

Reply:
@jenstrickland Please educate me: if `aria-hidden` is `true` is the `role` still used at all?

Reply:
Yes, generally we would use both. My 🤠  pal John Foliot explains it better (and more entertainingly) than I can: http://john.foliot.ca/aria-hidden/

Reply:
Thanks for the link to that, @jenstrickland. It looks like `aria-hidden="true"` had issues in IE 8 and 9 and a 2011 version of NVDA in a very old version of Firefox (but even that version of NVDA worked with Chrome). So it seems like there were major bugs with `aria-hidden="true"` back in 2012 that were hopefully resolved some time ago? A quick search for issues with `aria-hidden ie11` doesn't turn up much of anything except [one sketchy SO post](https://stackoverflow.com/questions/36873907/nvda-ignoring-aria-hidden-true-in-ie-11-and-10) and [one super edge-case-y bug with NVDA 2017](https://github.com/nvaccess/nvda/issues/7585).

I know we have a list of browsers that we officially support. Do we have the same for screen readers?

Reply:
We usually use both `aria-hidden="true"` and `role="presentation"` in our work. 

In testing we use these combinations of browser + screen reader:
- IE11 + JAWS
- Firefox + NVDA
- Safari + VoiceOver

There are cases where we might not consider these icons as presentation, and instead use an `aria-label`. For these checkmarks, that could be an option, too, with `aria-label="Checked"`. However, I think the page is clear if the icon is suppressed.